static/app/views/alerts/rules/issue/addIntegrationRow.tsx

```diff
@@ -0,0 +1,116 @@
+import {useCallback, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+import type {IntegrationProvider, Project} from 'sentry/types';
+import type {Organization} from 'sentry/types/organization';
+import useApi from 'sentry/utils/useApi';
+import {AddIntegrationButton} from 'sentry/views/settings/organizationIntegrations/addIntegrationButton';
+
+type Props = {
+  closeModal: () => void;
+  integrationSlug: string;
+  organization: Organization;
+  project: Project;
+  setHasError: (boolean) => void;
+};
+
+function AddIntegrationRow({
+  integrationSlug,
+  organization,
+  project,
+  closeModal,
+  setHasError,
+}: Props) {
+  const [provider, setProvider] = useState<IntegrationProvider | null>(null);
+
+  const api = useApi();
+  const fetchData = useCallback(() => {
+    if (!integrationSlug) {
+      return Promise.resolve();
+    }
+
+    const endpoint = `/organizations/${organization.slug}/config/integrations/?provider_key=${integrationSlug}`;
+    return api
+      .requestPromise(endpoint)
+      .then(integrations => {
+        setProvider(integrations.providers[0]);
+      })
+      .catch(() => {
+        setHasError(true);
+      });
+  }, [integrationSlug, api, organization.slug, setHasError]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData]);
+
+  if (!provider) {
+    return null;
+  }
+
+  const {metadata} = provider;
+  const size = 'sm' as const;
+  const priority = 'primary' as const;
+
+  const buttonProps = {
+    style: {margin: 0},
+    size,
+    priority,
+    'data-test-id': 'install-button',
+    organization,
+  };
+
+  // TODO(Mia): disable button if user does not have necessary permissions
```

Comment:
Just to make sure we're aligned, if user doesn't have permission we render instead of install button the request button.